docs_italia_convertitore_web/static/js/docs_italia_convertitore.js

```diff
@@ -0,0 +1,56 @@
+Dropzone.autoDiscover = false;
+$(document).ready(function() {
+    $(function () {
+        $('[data-toggle="popover"]').popover()
+    });
+    var validator = $('#converterForm').validate({
+        rules: {
+            email: {
+                required: true,
+                email: true
+            },
+            file: {
+                required: true
+            }
+        },
+        errorPlacement: function(error, element) {
+            error.appendTo('.errors');
+        }
+    });
+    new Dropzone('#fileInput', {
+        url: "#",
+        maxFiles:1,
+        acceptedFiles: "application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/vnd.oasis.opendocument.text,.md",
+        autoProcessQueue: false,
+        init: function () {
+            var myDropzone = this;
+            $("#submit").click(function (e) {
+                e.preventDefault();
+                myDropzone.processQueue();
+                if(!validator.form()) {
+                    alert("E' necessario fornire un indirizzo email");
+                }
+            });
+            myDropzone.on('success', function () {
+                window.location.href = "{% url 'docs_italia_convertitore:conversion-started' %}";
```

Comment:
Muovendo questa url in un js separato questa non funziona. Potremmo impostare questa variabile nel template converter_index.html e usarla nel js

Reply:
Azz, non l'avevo notato. Sistemo subito, grazie!